### PR TITLE
Add logger

### PR DESCRIPTION
**Pull request by ovekyc**
How about add logger to our code??

### CODE REVIEW

**Comment by festinalente91** [comment]
I think it's a great idea. And integrating with [AWS CloudWatch](https://aws.amazon.com/ko/cloudwatch/)
seems interesting feature.

**Comment by ochanje210** [comment]
having our own logger is actually a very good idea

**Comment by ovekyc** [comment]
LGTM

File: src/server/index.js [hunk 1]
```diff
@@ -1,66 +1,78 @@
...
-    console.log('Mock data was successfully stored.');
-    /* eslint-enable */
+    logger.info('Mock data was successfully stored.');
+  });
+
+  /*
+    config winston logger
+    @TODO it would be better if you configure winston with log monitoring service(eg. cloudwatch)
+  */
+  const logFileName = `created-logfile.log`;
```
Comment (ovekyc) [inline src/server/index.js]: how about make directory for log and named log file with date?
like
`log/${new Date().toISOString()}.log`

**Comment by ovekyc** [comment]
LGTM

**Comment by hewonjeong** [comment]
LGTM

**Pull request merged**